<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ID Genrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="Tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="setupTab">
       <property name="maximumSize">
        <size>
         <width>1125</width>
         <height>715</height>
        </size>
       </property>
       <attribute name="title">
        <string>Setup</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="textBoxIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>100</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">margin: auto 0;
                                                    text-align: center;
                                                    height: auto;
                                                    border: 1px solid grey;
                                                    background-color: white
                                                </string>
              </property>
              <property name="text">
               <string>Text Box</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="template_img">
              <property name="maximumSize">
               <size>
                <width>800</width>
                <height>600</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>MGI_Blank Lvl1.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="text">
             <string>Upload Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generateIDButton">
            <property name="text">
             <string>Generate IDs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Data</string>
       </attribute>
        <item>
         <widget class="QToolBox" name="toolBox">
          <widget class="QWidget" name="toolBoxPage1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>914</width>
             <height>617</height>
            </rect>
           </property>
           <attribute name="label">
            <string>TextBox1</string>
           </attribute>
           <widget class="QFontComboBox" name="fontComboBox">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>9</y>
              <width>154</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QPlainTextEdit" name="textBox2_Data_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>1211</width>
              <height>87</height>
             </rect>
            </property>
           </widget>
           <zorder>textBox2_Data_3</zorder>
           <zorder>fontComboBox</zorder>
          </widget>
          <widget class="QWidget" name="toolBoxPage2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>914</width>
             <height>617</height>
            </rect>
           </property>
           <attribute name="label">
            <string>TextBox2</string>
           </attribute>
           <widget class="QPlainTextEdit" name="textBox2_Data">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>1211</width>
              <height>87</height>
             </rect>
            </property>
           </widget>
           <widget class="QFontComboBox" name="fontComboBox_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>226</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>